tests/python/test_simt.py

```diff
@@ -24,8 +26,24 @@ def test_unique():
 
 @test_utils.test(arch=ti.cuda)
 def test_ballot():
-    # TODO
-    pass
+    a = ti.field(dtype=ti.u32, shape=32)
+    b = ti.field(dtype=ti.u32, shape=32)
+
+    @ti.kernel
+    def foo():
+        ti.loop_config(block_dim=32)
+        for i in range(32):
+            a[i] = ti.simt.warp.ballot(b[i])
+
+    key = 0
+    for i in range(32):
+        b[i] = random.randint(1, 100) % 2
```

Comment:
nit, can you change this to utilize ti.random()? ;/

Reply:
My two cents: use constant numbers instead of random numbers to avoid random test behaviors.